openlibrary/templates/lib/markdown.html

```diff
@@ -1,75 +1,75 @@
         <p>$_("Formatting marks should envelope the effected text, for example:")
-        &lt;strong&gt;<strong>This text is bold</strong>&lt;strong&gt; OR **<strong>This text is also bold</strong>**</p>
+        &lt;strong&gt;<strong>$_("This text is bold")</strong>&lt;strong&gt; OR **<strong>$_("This text is also bold")</strong>**</p>
         <table class="help" id="format">
             <thead>
                 <tr>
-                    <th>The format</th>
-                    <th class="teal">HTML</th>
-                    <th class="blue">Markdown</th>
+                    <th>$_("The format")</th>
+                    <th class="teal">$_("HTML")</th>
+                    <th class="blue">$_("Markdown")</th>
                 </tr>
             </thead>
             <tbody>
                 <tr>
-                    <td><em>Italic</em></td>
-                    <td><span class="teal">&lt;em&gt; &lt;/em&gt;</span> (preferred) or <span class="teal">&lt;i&gt; &lt;/i&gt;</span></td>
-                    <td><span class="blue">* *</span> or <span class="blue">_ _</span> (single underscores)</td>
+                    <td><em>$_("Italic")</em></td>
+                    <td><span class="teal">&lt;em&gt; &lt;/em&gt;</span>$_(" (preferred) or ")<span class="teal">&lt;i&gt; &lt;/i&gt;</span></td>
+                    <td><span class="blue">* *</span>$_(" or ")<span class="blue">_ _</span>$_(" (single underscores)")</td>
                 </tr>
                 <tr>
-                    <td><strong>Bold</strong></td>
-                    <td><span class="teal">&lt;strong&gt; &lt;/strong&gt;</span> (preferred) or <span class="teal">&lt;b&gt; &lt;/b&gt;</span></td>
-                    <td><span class="blue">** **</span> or <span class="blue">__ __</span> (double underscores)</td>
+                    <td><strong>$_("Bold")</strong></td>
+                    <td><span class="teal">&lt;strong&gt; &lt;/strong&gt;</span>$_(" (preferred) or ")<span class="teal">&lt;b&gt; &lt;/b&gt;</span></td>
+                    <td><span class="blue">** **</span>$_(" or ")<span class="blue">__ __</span>$_(" (double underscores)")</td>
                 </tr>
                 <tr>
-                    <td>Line<br/>Break</td>
+                    <td>$:_('Line<br/>Break')</td>
                     <td><span class="teal">&lt;br/&gt;</span></td>
-                    <td>Insert two spaces at the end of the line.</td>
+                    <td>$_("Insert two spaces at the end of the line.")</td>
                 </tr>
                 <tr>
-                    <td><blockquote>Blockquote</blockquote></td>
+                    <td><blockquote>$_("Blockquote")</blockquote></td>
                     <td><span class="teal">&lt;blockquote&gt; &lt;/blockquote&gt;</span></td>
-                    <td>Precede copy with <span class="blue">&gt;</span> </td>
+                    <td>$_("Precede copy with ")<span class="blue">&gt;</span> </td>
```

Comment:
If it's not too complicated we do want the html entites inside here
```suggestion
                    <td>$:_('Precede copy with <span class="blue">&gt;</span> ')</td>
```